index.bs

```diff
@@ -1956,15 +1957,15 @@ To <dfn>parse trigger specs</dfn> given a [=map=] |map|, a [=moment=]
         1. If |reportWindows| is an error, return it.
         1. Set |spec|'s [=trigger spec/event-level report windows=] to
             |reportWindows|.
-    1. Let |summaryWindowOperator| be "<code>[=summary window operator/count=]</code>".
+    1. Set |spec|'s [=trigger spec/summary window operator=] to
```

Comment:
Sorry, I meant that we could do this as part of the `Let |spec| be a new [=trigger spec=]...` step above.

Reply:
I'm assuming I should add the default values for these fields to "if map['trigger_specs'] does not exist" as well right?

Reply:
I removed the fields as well as the parsing implementation within the trigger spec algorithm. Please lmk if you meant something else, thanks!